src/style.css

```
@@ -1,3 +1,94 @@
+/* Reset */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Base font */
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+/* CSS Variable */
+:root {
+  --color-blue: #00acdc;
+}
+
+/* Header */
+header {
+  max-width: 100vw;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+  height: 60px;
+  color: #fff;
+}
+
+/* Logo */
+.logo-link {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  padding: 0;
   margin: 0;
 }
+
+/* Navigation */
+.navigation {
+  flex-shrink: 0;
+}
+
+/* List */
+.nav-links {
+  display: flex;
+  height: 64px;
+  align-items: center;
+  list-style: none;
+}
+
+/* Items */
+.nav-links li + li {
+  margin-left: 20px; /* spacing between inner items only */
+}
+
+/* Links */
+.nav-links a {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  text-align: center;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  position: relative;
+  transition: color 0.3s;
+  padding: 0;
+}
+
+/* Hover */
+.nav-links a:hover {
+  color: var(--color-blue);
+}
+
+/* Active */
+.nav-links a.is-active {
+  color: var(--color-blue);
+  margin: 0;
+  position: relative;
+}
+
+.nav-links a.is-active::after {
```

Use only simplify selectors
```suggestion
.is-active::after {
```